types/index.d.ts

```diff
@@ -75,7 +75,7 @@ export interface ConfigOptions {
 	encoding?: BufferEncoding;
 	layoutsDir?: string;
 	partialsDir?: string|string[]|PartialsDirObject|PartialsDirObject[];
-	defaultLayout?: string;
+	defaultLayout?: string|boolean;
```

Comment:
I don't think we want to allow `true` but false should be valid.

```suggestion
	defaultLayout?: string|false;
```